test/dd-trace-zipkin.spec.js

```diff
@@ -2,7 +2,6 @@
 
```

Comment:
There's some side effect of the core tests that's carrying over by the time it gets to the end of the tests. The easiest fix was to reorder it. I didn't want to spend any more time figuring it out, since the reason might become more apparent with some more familiarity with the code.

@rmfitzpatrick @sm7x since you have been in this longer, is there anything that stands out as an obvious cause?

Reply:
Haven't run into this and not seeing anything of pressing value, so this seems fine.